### PR TITLE
Created a new scenaio for bundle-resolver

### PR DESCRIPTION
**Pull request by siddardh-ra**
Created a new scenario for bundle resolver 

Here I'm referencing this public repository in `quay.io` which I created for testing : https://quay.io/repository/sira_org/pipelines_perftesting/tag/task

JIRA :https://issues.redhat.com/browse/SRVKP-6099

### CODE REVIEW

File: tests/scaling-pipelines/scenario/bundle-resolver/pipeline.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+spec:
+  params: []
+  tasks:
+    {% if resolver_type == "bundle" %}
+    - name: echo-{{ step_count }}-step
+      taskRef:
+        resolver: bundles
+        params:
+        - name: bundle
+          value: quay.io/sira_org/pipelines_perftesting@sha256:3c1d7bfe5a44a8f8964a37f82c2dc167c264616d2fb6aa21d4364afed25076bc
```
[the remark this answers is not in the document]
Comment (siddardh-ra) [reply tests/scaling-pipelines/scenario/bundle-resolver/pipeline.yaml.j2:14]: Hey @rh-rahulshetty ,
>Also we could think of placing the bundle in a public repository for easier access.
Yes. It was already a [public repo](https://quay.io/repository/sira_org/pipelines_perftesting/tag/task), so it doesn't require any secrets to access the files in that bundle.

**Review by jhutar — APPROVED** [review]
/lgtm /approve

**Review by khrm — COMMENTED** [review]
/lgtm

**Pull request merged**